charts/rancher-monitoring-crd/105.1.0-rc1+up61.3.2/values.yaml

```diff
@@ -0,0 +1,17 @@
+# Default values for rancher-monitoring-crd.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  cattle:
+    psp:
+      enabled: false
+    systemDefaultRegistry: ""
+
+image:
+  repository: rancher/shell
+  tag: v0.2.1
```

Comment:
Technically this should be a v0.3 - but I haven't produced that for shell yet.

Since the shell project is in-flight it's a bit odd too, as if helm is not needed it should use https://github.com/rancher/kuberlr-kubectl instead and could use the `v3` branch from there.

Mostly this is just an aside mention and nothing actionable yet.

Reply:
Ok sick, I'll leave the shell bump for a later version then!